.github/workflows/create-release-pr-to-develop.yml

```diff
@@ -12,15 +12,6 @@ jobs:
         with:
           token: ${{ secrets.MERGE_TOKEN }}
           fetch-depth: 0
-      - name: enable merge commits
-        uses: octokit/request-action@v2.x
-        with:
-          route: PATCH /repos/{owner}/{repo}
-          owner: aws-solutions
-          repo: solution-spark-on-aws
-          allow_merge_commit: true
-        env:
-          GITHUB_TOKEN: ${{ secrets.MERGE_TOKEN }}
       - name: Import GPG key
```

Comment:
is there actually a commit somewhere in here that's being signed?

Reply:
the commit is merging release into a intermediate rebase branch